xarray/tests/test_backends.py

```diff
@@ -3223,6 +3231,10 @@ def test_utm(self):
         with create_tmp_geotiff() as (tmp_file, expected):
             with xr.open_rasterio(tmp_file) as rioda:
                 assert_allclose(rioda, expected)
+                assert rioda.attrs['scales'] == (1.0, 1.0, 1.0)
```

Comment:
I don't see where these attributes are being set. How is this working?

Reply:
I added scales attribute in the first commit of this PR.

https://github.com/pydata/xarray/pull/3013/files#diff-6364b203943c799516f9ecba19e1b119R306

Reply:
No I mean for the tests. Where is (1.0, 1.0, 1.0) being set for scales? I see descriptions and units being set.

Reply:
Geotiff bands have a default value 1.0 for scale.

Reply:
Ah ok. Can you either add a comment saying that or set custom values for the test so that people looking over the code in the future are not confused? (I think I prefer the latter)

Apologies for the delay. With that, I think this should be good to merge.